our-experiments.md

```diff
@@ -0,0 +1,55 @@
+# Our Experiments
+
+As we mentioned [in the presentation document](join-us.md), **_Design all the things_** is an initiative to experiment a _Domain Driven Design_ approach to build software. We try to use and learn principles, practices and workshops which could help us better understand the problem and the corresponding domain, with which _**Releasy**_ deals.
```

Comment:
`with which _**Releasy**_ deals` => which _**Releasy**_ deals with  ?

(Watch out: my english is not really good 😆 )